apps/kitchen-sink/src/App.test..js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import "./App.css";
+import mondaySdk from "monday-sdk-js";
+import "monday-ui-react-core/dist/main.css";
+import AttentionBox from "monday-ui-react-core/dist/AttentionBox.js";
+
+const monday = mondaySdk();
+
+class App extends React.Component {
```

Comment:
Some components are functional, some are classes - let's be consistent and use functional components only